scripts/setup/create_multinode.sh

```diff
@@ -100,31 +96,24 @@ function setup_master() {
 
     clone_loader $MASTER_NODE
 
-    MN_CLUSTER="./vhive/scripts/cluster/create_multinode_cluster.sh ${OPERATION_MODE}"
+    server_exec $MASTER_NODE 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
```

Comment:
yq version must be defined in one place, current it's used here and in another sh file.